core/licenses/jna-4.4.0.jar.sha1

```diff
@@ -1 +1 @@
-6edc9b4514969d768039acf43f04210b15658cd7
\ No newline at end of file
+d85e9ceee4d97e738524a231687ff28446376c97
```

Comment:
I've implemented this as part of elastic/jna-build#1 in elastic/jna-build@c575bfb9569608a4ed9153981f166add9ecc0b65. Please review that particular change over there. I'll bump the version and SHA here.